src/hclib.c

```diff
@@ -355,9 +369,9 @@ void forasync2D_flat(void * forasync_arg) {
             printf("Scheduling Task %d %d\n",low1,high1);
```

Comment:
Same comment, see above. Things I add are highlighted in green, things I delete are highlighted in red. This isn't highlighted in either, so it is unchanged in this diff.